generated/api.github.com.json

```diff
@@ -79,56 +52,68 @@
       "write": []
     },
     "actions": {
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-actions",
+      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#actions",
       "read": [
         "GET /repos/{owner}/{repo}/actions/artifacts",
         "GET /repos/{owner}/{repo}/actions/artifacts/{artifact_id}",
         "GET /repos/{owner}/{repo}/actions/artifacts/{artifact_id}/zip",
         "GET /repos/{owner}/{repo}/actions/cache/usage",
         "GET /repos/{owner}/{repo}/actions/jobs/{job_id}",
         "GET /repos/{owner}/{repo}/actions/jobs/{job_id}/logs",
+        "GET /repos/{owner}/{repo}/actions/runners",
+        "GET /repos/{owner}/{repo}/actions/runners/downloads",
+        "GET /repos/{owner}/{repo}/actions/runners/{runner_id}",
+        "GET /repos/{owner}/{repo}/actions/runners/{runner_id}/labels",
         "GET /repos/{owner}/{repo}/actions/runs",
         "GET /repos/{owner}/{repo}/actions/runs/{run_id}",
         "GET /repos/{owner}/{repo}/actions/runs/{run_id}/artifacts",
         "GET /repos/{owner}/{repo}/actions/runs/{run_id}/jobs",
         "GET /repos/{owner}/{repo}/actions/runs/{run_id}/logs",
         "GET /repos/{owner}/{repo}/actions/workflows",
         "GET /repos/{owner}/{repo}/actions/workflows/{workflow_id}",
-        "GET /repos/{owner}/{repo}/actions/workflows/{workflow_id}/runs"
+        "GET /repos/{owner}/{repo}/actions/workflows/{workflow_id}/runs",
+        "GET /repos/{owner}/{repo}/interaction-limits",
```

Comment:
why is this under actions, that doesn't seem right

Reply:
something is definitely still funky, this is not ready, I'll need to have a look next week